src/rebase.js

```diff
@@ -228,6 +228,17 @@ module.exports = (function(){
     }
   };
 
+  function _update(endpoint, options){
+    _validateEndpoint(endpoint);
+    optionValidators.data(options);
+    var ref = new Firebase(`${baseUrl}/${endpoint}`);
```

Comment:
@foffer sorry meant line 234 in src. needs update to be compatible with firebase 3

```javascript 
var ref = firebase.database().ref(endpoint);
```